frontend/src/components/sequenceEditOptionsTab/QuillToolBar.vue

```diff
@@ -128,3 +126,14 @@ onMounted(() => {
     ?.toggleAttribute('selected', true);
 });
 </script>
+
+<style scoped>
+.active-indicator > .ql-active {
```

Comment:
change to `<style scoped lang="postcss"> and then use `@apply class` to apply tailwind classes for styling